scss/_patterns_navigation.scss

```diff
@@ -283,28 +282,32 @@ $lightness-threshold: 70;
 
 @mixin vf-navigation-theme(
   // color for navigation separator on small screens
-  $color-navigation-separator,
+    $color-navigation-separator,
```

Comment:
Prettier is complaining on something, possibly these spaces.

It was changed in recent version of prettier, so make sure you have up to date dependencies.

Reply:
these are added automatically on save. I did dotrun clean and dotrun, is that enough to refresh dependancies?